README.md

```diff
@@ -30,8 +30,10 @@ Navigate through the web page to find the information you need. Use the links in
 
 
 ### Deployment
-  Project reside in this Repo. You can clone the repo or download the files manually to your local machine.
+  Project reside in this Repo and is hosted on netlify. You can clone the repo or download the files manually to your local machine.
+  below is the online link for this website:
```

Comment:
- PLease as stated in the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/5_deploy.md), you are to make use of GitHub pages and not netlify for this project, please make use of GitHub pages to deploy your webpage.

Reply:
### Fixes applied according to the requirement